protos/feast/third_party/grpc/connector/Connector.proto

```diff
@@ -0,0 +1,34 @@
+syntax = "proto3";
```

Comment:
I wonder if this connector interface shouldnt rather be within `serving`. Third party is normally for external protos that we pull in. This is still our proto.